kore/src/Kore/Repl/Data.hs

```diff
@@ -267,11 +278,11 @@ helpText =
     \label <+l> [n]                        add a new label for a node\
                                            \ (defaults to current node)\n\
     \label <-l>                            remove a label\n\
-    \try <a|c><num>                        attempts <a>xiom or <c>laim at\
-                                           \ index <num>\n\
-    \tryf <a|c><num>                       attempts <a>xiom or <c>laim at\
-                                           \ index <num> and if successful, it\
-                                           \ will apply it.\n\
+    \try <a|c><num>|<name>                 attempts <a>xiom or <c>laim at\
```

Comment:
Maybe some parentheses would help here and below `(<a|c><num>)|<name>`, to make it clearer?